Contributing.md

```diff
@@ -118,8 +118,8 @@ nominate someone to take their place.
 TC members will be added as admin's on the Github orgs, npm orgs, and other resources as
 necessary to be effective in the role.
 
-To remain "active" a TC member should have participation within the last 6 months and miss
-no more than three consecutive TC meetings. Members who do not meet this are expected to step down.
+To remain "active" a TC member should have participation within the last 12 months and miss
```

Comment:
I think we can hold off for now, with a fast follow in Season 0 to firm this up.

The penalty for inactivity requires a PR to be opened to remove someone who has not met the attendance criteria, and does not choose to resign. (it is also not spelled out WHO can open or approve that PR hehe)

I am confident that the current TC will not inactive someone if they miss the next couple meetings. 

Churn in these policies need to be avoided lest we hinder their gravity. Let's make sure that the next change we make to this clause is one that balances our goals here (encouraging activity and showing up) as well as is something we won't have to touch again for a long time unless something drastic changes.

Reply:
"To promote active participation, TC members must attend a minimum of 75% of official monthly meetings annually. To account for time zone challenges, members may 'officially attend' by submitting detailed comments on the meeting agenda at least 12 before the meeting. This will count as attendance for maintaining active status.

Ad hoc working sessions are vital for progress but are not mandatory and do not count towards the monthly meeting attendance requirement. **Official decisions requiring consensus or vote are reserved for monthly meetings, except under urgent circumstances, where a predefined exception process will be followed.**"

The bolded part here is something that we can omit, but should definitely be broached and put into writing. Im guessing we could strike that rule, and would have to depending on how "official decisions" are defined. Is merging a PR an official decision? Prolly not. Accepting a new TC member def is though, as would be emptying the (nonexistent) bank account.

Reply:
Oh I like this more detailed version!